client/packages/common/src/ui/components/inputs/DateTimePickers/DateTimePickerInput/DateTimePickerInput.tsx

```diff
@@ -50,12 +73,14 @@ export const DateTimePickerInput = ({
   dateAsEndOfDay,
   disableFuture,
   error,
+  errorText,
   required,
   textFieldSx: inputSx,
   slotProps,
   ...props
 }: Omit<DateTimePickerProps<true>, 'onChange'> & {
```

Comment:
should this one also use the new type above?

Reply:
Right, yeah, looks like somewhere in the merging I ended up with a double definition.

047e7c7771